includes/definitions/svos.yaml

```diff
@@ -0,0 +1,9 @@
+os: svos
+text: 'Hitachi Storage Virtualization Operating System (SVOS)'
+type: storage
+icon: hds
+mib_dir:
+    - hitachi
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.116.3.11.4.1.1
```

Comment:
I set it to `.1.3.6.1.4.1.116.3.` just to be safe. Hitachi has quite a bit of other product that probably does not run svos